src/Page/PageCanvas.jsx

```diff
@@ -118,7 +120,7 @@ export class PageCanvasInternal extends PureComponent {
     const renderContext = {
       annotationMode: renderForms ? ANNOTATION_MODE.ENABLE_FORMS : ANNOTATION_MODE.ENABLE,
       get canvasContext() {
-        return canvas.getContext('2d', { alpha: false });
+        return canvas.getContext('2d', readFrequently ? { willReadFrequently: true } : { alpha: false });
```

Comment:
Should this really be one or the other? `alpha: false` does resolve a plethora of rendering issues